css/style.css

```
@@ -0,0 +1,997 @@
+@import url("https://fonts.bunny.net/css?family=dm-sans:300,700|space-grotesk:300,400,700");
+@import url("https://use.fontawesome.com/releases/v6.5.2/css/all.css");
+
+/* @link https://utopia.fyi/type/calculator?c=320,16,1.414,1680,16,1.414,5,2,&s=0.75|0.5|0.25,1.5|2|3|4|6,s-l&g=s,l,xl,12 */
+
+:root {
+  --turquoise: #209e9b;
+  --midturquoise: #1B8381;
+  --darkturquoise: #004746;
+  --pink: #f03278;
+  --orange: #f68243;
+  --yellow: #faae30;
+  --green: #4fc066;
+  --white: #fff;
+  --dark-grey: #4b4b4b;
+  --light-grey: #696969;
+  --8px: .5rem;
+  --12px: .75rem;
+  --16px: 1rem;
+  --20px: 1.25rem;
+  --24px: 1.5rem;
+  --48px: 3rem;
+  --96px: 6rem;
+  --step--1: clamp(.8rem,0.6706rem + 0.1471vw, 1rem);
+  --step-0: clamp(1rem, 0.9706rem + 0.1471vw, 1.125rem); /* Base */
+  --step-1: clamp(1.2rem, 1.1515rem + 0.2426vw, 1.4063rem);
+  --step-2: clamp(1.44rem, 1.3652rem + 0.3739vw, 1.7578rem);
+  --step-3: clamp(1.728rem, 1.6176rem + 0.5521vw, 2.1973rem);
+  --step-4: clamp(2.0736rem, 1.9153rem + 0.7917vw, 2.7466rem);
+  --step-5: clamp(2.4883rem, 2.266rem + 1.1117vw, 3.4332rem);
+}
+
+/* CSS Reset */
+
+/* Box sizing rules */
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+/* Prevent font size inflation */
+html {
+  -moz-text-size-adjust: none;
+  -webkit-text-size-adjust: none;
+  text-size-adjust: none;
+}
+
+/* Remove default margin in favour of better control in authored CSS */
+body, h1, h2, h3, h4, p,
+figure, blockquote, dl, dd {
+  margin-block-end: 0;
+}
+
+/* Remove list styles on ul, ol elements with a list role, which suggests default styling will be removed */
+ul[role='list'],
+ol[role='list'] {
+  list-style: none;
+}
+
+/* Set shorter line heights on headings and interactive elements */
+h1, h2, h3, h4,
+button, input, label {
+  line-height: 1.1;
+}
+
+/* Balance text wrapping on headings */
+h1, h2,
+h3, h4 {
+  text-wrap: balance;
+}
+
+/* Make images easier to work with */
+img,
+picture {
```

```suggestion
picture,
video {
```

I guess this should fix the big video at the end of https://servo.org/servo-redesign/blog/2024/02/28/gamepads-font-fallback-space-jam/

---

Yeah it does fix it.